lms/static/js/commerce/views/receipt_view.js

```diff
@@ -3,7 +3,7 @@
  */
 var edx = edx || {};
 
-(function ($, _, _s, Backbone) {
+(function ($, _, Backbone, interpolate_text) {
```

Comment:
Thanks for removing the Underscore.string usages. This wasn't clear, but ``interpolate_text`` is being deprecated in favor of ``StringUtils.interpolate`` and ``HtmlUtils.interpolateHtml``. In your case, use ``edx.StringUtils.interpolate`` as this code is not using RequireJS.

See the safe template guidelines here:

http://edx-developer-guide.readthedocs.io/en/latest/conventions/safe_templates.html